XCUITest/CopyTest.swift

```diff
@@ -39,6 +43,6 @@ class CopyTest: BaseTestCase {
             return
         }
         
-        XCTAssert(text == "https://www.google.com/")
+        XCTAssert(text == "https://m.facebook.com/")
```

Comment:
nit: we should probably use an internal website (something Mozilla based) rather than Facebook since we don't need to test anything about it being a mobile site.

Reply:
I changed from Google since Google now tacks on some misc. to the end of their URL that obscures it. I tried Mozilla's website, but that URL changes based on locale, so that's why I decided Facebook (just because we use it in other places for testing and it's seemingly stable.) Is there another internal website that you know of that doesn't change its URL?

Reply:
I believe there is a localhost file we could use, but if that doesn't work Facebook is fine!